src/seedu/addressbook/AddressBook.java

```diff
@@ -210,13 +210,17 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        userInterface();
+    }
+
+	public static void userInterface() {
+		while (true) {
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)